NRediSearch/Schema.cs

```diff
@@ -95,7 +114,18 @@ public Schema AddGeoField(string name)
         /// <returns>the schema object</returns>
         public Schema AddNumericField(string name)
         {
-            Fields.Add(new Field(name, FieldType.Numeric));
+            Fields.Add(new Field(name, FieldType.Numeric, false));
+            return this;
+        }
+
+        /// <summary>
+        /// Add a numeric field that can be sorted on
+        /// </summary>
+        /// <param name="name">the field's name</param>
+        /// <returns>the schema object</returns>
+        public Schema AddSortableNumericField(string name)
```

Comment:
this *could* also be done by tweaking `AddNumericField(string name)` to be `AddNumericField(string name, bool sortable = false)` - i.e. so that it defaults to the old non-sortable behavior. Technically that's a breaking change (the signature is no longer compatible), but I *suspect* that's OK at this point in time. It wouldn't be OK in fully established lib without a major (semver) incr.